### DISCUSSION TITLE
Help me Please!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#80364] [Programming Help]
### Body

So hi I'm @Yuheshpandian. I had joined GitHub on the year 2022. But only in this month I started to be consistent in it. I'm actually facing a problem that most programmers face in GitHub, the problem of reaching people. My projects are performing poor as it is not reaching everyone due to popularity issues. So I actually want all of you to look out my projects, review them, add your suggestions on them, contribute and star them if you like. This would help me a lot, please help me.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Sachintha-Samarathunga** [comment]
Response #1

**Comment by Sachintha-Samarathunga** [comment]
Response #2

**Comment by ccrisc** [comment]
Sure!👋

**Comment by jane-does-coding** [comment]
Happy to help, I have the same issue right now, I am trying to put more projects on so if you can please follow me and star what I have.

**Comment by clyde-dot** [comment]
Hey @Yuheshpandian! It's awesome that you're diving into GitHub more consistently now. The struggle to get visibility for projects is real, but don't worry—we've all been there! Sharing your projects in communities related to their themes or technologies can help. Also, giving feedback and contributing to others' projects often leads to them checking out yours too. Keep up the great work!
